src/vos/vos_dtx.c

```diff
@@ -2866,6 +2866,9 @@ vos_dtx_act_reindex(struct vos_container *cont)
 			dae->dae_need_release = 1;
 			D_INIT_LIST_HEAD(&dae->dae_link);
 
+			if (DAE_FLAGS(dae) & DTE_PARTIAL_COMMITTED)
+				DAE_REC_CNT(dae) = 0;
```

Comment:
The DAE_REC_CNT is supposed to be zero when the partial commit flag is set, so It's better to throw an error message when we hit non-zero record count for partial commit entry.

Reply:
At that time, no data corruption happened yet, so such error or warning may misguide the user.